service/worker/failovermanager/workflow_test.go

```diff
@@ -360,7 +368,7 @@ func (s *failoverWorkflowTestSuite) TestShouldFailover() {
 				},
 			},
 			sourceCluster: "c2",
-			expected:      true,
+			expected:      false,
```

Comment:
Why is the test result changed?

Reply:
it was wrong to begin with, the cluster attributes were not being skipped in parity with the clustersByRegion